FinSearchUnified/Tests/Subscriber/FrontendTest.php

```diff
@@ -4,14 +4,20 @@
 
 use Enlight_Controller_Action;
 use Enlight_Controller_Request_RequestHttp as RequestHttp;
+use Enlight_Controller_Response_ResponseHttp;
 use Enlight_Event_EventArgs;
 use Enlight_Hook_HookArgs;
 use Shopware\Components\Test\Plugin\TestCase;
 use Shopware_Controllers_Widgets_Listing;
-use Enlight_Controller_Response_ResponseHttp;
 
 class FrontendTest extends TestCase
 {
+    public static function tearDownAfterClass()
+    {
+        parent::tearDownAfterClass();
+        $_GET = [];
```

Comment:
Move this to the `tearDown` method, just in case it would affect multiple tests.